src/Resources/views/tailwind_2/_table.html.twig

```diff
@@ -302,5 +302,26 @@
             {% endfor %}
         {% endfor %}
         </tbody>
+        {% block footer_columns %}
+            {% if table.footerColumns is not null %}
+                <tfoot class="whatwedo_table-foot whatwedo-utility-paragraph">
+                <tr class="whatwedo_table-row border-t border-neutral-300 bg-neutral-200 text-base font-medium text-neutral-600 text-left tracking-wide align-top">
+                    {% for footerColumn in table.footerColumns %}
+                        {% set attr = footerColumn.option('attributes')|default([])|filter((k,i) => k != 'class') %}
+                        <td
+                            class="px-3 py-2 {{ footerColumn.option('attributes')['class'] ?? '' }}"
+                            {{ attr|map((value, attr) => "#{attr}=\"#{value}\"")|join(' ')|raw }}
```

Comment:
More of an info: These can become troubling with certain Twig formatters. Instead of mapping and filtering in Twig, it would be more resilient to prepare the data for the view in PHP already.

Reply:
Symfony does it in twig as well, a bit different though: https://github.com/symfony/symfony/blob/7.2/src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig#L477

Reply:
@tuxes3 It's just the "inline functions" for these Twig Functions (filter(function), map(function)) that are the troubling part for formatters. Not the fact that something is mapped or filtered in the template.

Their version is more verbose but doesn't fall into the troubling case for formatters.